cg/models/orders/samples.py

```diff
@@ -16,6 +16,22 @@
 from cg.store import models
 
 
+class OptionalIntValidator:
+    @classmethod
+    def str_to_int(cls, v: str) -> Optional[int]:
+        if v in (None, ""):
+            return None
+        return int(v)
```

Comment:
```suggestion
        if v:
            return int(v)
        
```

Reply:
Python treats `0` as `False` and thus we can't do `if v:` in this case, otherwise a good suggestion to clean up the code

Reply:
str("0") is True. If this is a string validator, I dont see a problem with this

Reply:
Yes, that's right, as long as it is a string ant not an int

Reply:
I guess we could make sure it is a string from where we parse the json and excel but it might make more sense to have it as an int?

Reply:
fixed

Reply:
I forgot to push the commit, I'll move it to the next PR instead